src/src/targets.h

```diff
@@ -194,8 +194,8 @@ Credit to Jacob Walser (jaxxzer) for the pinout!!!
 #define GPIO_PIN_DEBUG_TX    PA9 // confirmed
 
 
-#define BUFFER_OE               PA5  //CONFIRMED
-#define BUFFER_OE_ACTIVE HIGH
+#define GPIO_PIN_RC_INVERT      PA5  //CONFIRMED
+#define GPIO_PIN_RC_INVERT_TX   HIGH
```

Comment:
Use 0 or 1 and later xor it.

Reply:
That is how it is in develop HEAD already, I did not write that I just renamed the define. I'll update it to use 0/1 pending naming of the define above.

Reply:
yeah, others need to be changed as well at some point :)